Compile error in user code when using defmt-log feature

When writing an application for rp2040 using this library I run into this compile error (which doesn't appear when building the library in its own project).
```
error[E0308]: mismatched types
   --> /home/val/.cargo/git/checkouts/embedded-sdmmc-rs-e7242a4f827ce914/7b15cf5/src/./inner/fat/volume.rs:598:17
    |
598 | /                 debug!(
599 | |                     "LFN Contents {=u8} {=u8} {=u8:02x} {=[?; 13]:#04x}",
600 | |                     start, sequence, csum, buffer
601 | |                 );
    | |                 ^
    | |                 |
    | |_________________expected `&u8`, found `&bool`
    |                   arguments to this function are incorrect
    |
    = note: expected reference `&u8`
               found reference `&bool`
```
I will do a quick PR to solve this, since changing the first `{=u8}` to `{=bool}` solves the problem on a smaller example.

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -27,5 +27,7 @@ jobs:
         toolchain: ${{ matrix.rust }}
     - name: Build
       run: cargo build --no-default-features --features ${{matrix.features}} --verbose
+      env:
+        DEFMT_LOG: debug
     - name: Run Tests
       run: cargo test --no-default-features --features ${{matrix.features}} --verbose

--- a/src/fat/volume.rs
+++ b/src/fat/volume.rs
@@ -594,7 +594,7 @@ impl FatVolume {
                 debug!("LFN Contents {start} {sequence} {csum:02x} {buffer:04x?}");
                 #[cfg(feature = "defmt-log")]
                 debug!(
-                    "LFN Contents {=u8} {=u8} {=u8:02x} {=[?; 13]:#04x}",
+                    "LFN Contents {=bool} {=u8} {=u8:02x} {=[?; 13]:#04x}",
                     start, sequence, csum, buffer
                 );
                 match (start, sequence, self) {

